engine/function/src/templates/Basic.ftl

```diff
@@ -131,6 +132,35 @@ public class Basic {
         return result;
     }
 
+    /**
+     * Returns the length of the input.
+     *
+     * @param values values.
+     * @return length of the input or the Deephaven null constant for null inputs.
+     */
+    static public <T> long len(T[] values) {
+        if (values == null) {
+            return NULL_LONG;
+        }
+
+        return values.length;
+    }
+
+    /**
+     * Returns the length of the input.
+     *
+     * @param values values.
+     * @return length of the input or the Deephaven null constant for null inputs.
+     */
+    @SuppressWarnings("rawtypes")
+    static public long len(LongSizedDataStructure values) {
+        if (values == null){
```

Comment:
one more space (between closing parenthesis and open brace) and I'll be happy :-)